app/helpers/template_helpers.py

```diff
@@ -178,13 +184,21 @@ def get_survey_config(
         theme=theme,
         language=language,
         base_url=base_url,
+        schema=schema,
     )
 
 
 def render_template(template: str, **kwargs: Union[str, Mapping]) -> str:
     language = get_locale().language
+    if session_store := get_session_store():
+        if session_data := session_store.session_data:
+            # pylint: disable=assigning-non-slot
+            schema = load_schema_from_session_data(session_data)
+    else:
+        schema = None
```

Comment:
- We need session store for both schema and expiry time, therefore, we should do them in the same place. 
- Also, at the moment, schema is undefined if session data doesn't exist, and would result in an error.
- The pylint ignore is redundant.

We can do something like this:
```py
    schema, session_expires_at = None, None
    if session_store := get_session_store():
        if session_data := session_store.session_data:
            schema = load_schema_from_session_data(session_data)

        if session_expiry := session_store.expiration_time:
            session_expires_at = session_expiry.isoformat()
```

The session expiry assignment on line 218 can then be removed.

Reply:
Ok, fixed, thanks.

Reply:
> The session expiry assignment on line 218 can then be removed.

Still needs doing.

Reply:
Misunderstood this one, ok removed this assignment now.